### PR TITLE
Update Readme.md that includes importJSON section

### LINKED ISSUE #732: Provide example of JSON style steps syntax in README
In the docs here: https://github.com/publiclab/image-sequencer#string-syntax

we describe how a steps object can be passed in:

```
sequencer.importJSON(obj) //Imports the given sequence of JSON steps into sequencer
```

But we don't describe the formatting. I think there is an example in the tests, we can link to and/or quote in the docs. We should have a brief section below the string syntax section in the docs showing an example of the object syntax!

Also, we should ensure `addSteps()` can also accept a command JSON object too, as in #731 -- that could be a nice follow-up!

### PR DESCRIPTION
**Pull request by Divy123**
Fixes #732 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` and `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by Divy123** [comment]
One part of the issue is solved.
Working towards other part.

**Comment by Divy123** [comment]
@publiclab/is-reviewers please review.

File: README.md [hunk 1]
```diff
@@ -535,6 +541,23 @@ Image Sequencer can also generate a string for usage in the CLI for the current
...
+sequencer.importJSON(obj)
+```
+It can be implemented the following way for example:
+
+```js
+sequencer.importJSON([
+    { name: 'blur', options: {} }
+  ]);
+  ```
+  where name is the name of step to be added, options object can be the one used to provide various params to the sequencer which can customise the default ones.
```
[the remark this answers is not in the document]
Comment (Divy123) [reply README.md]: Sure thanks!

**Comment by Divy123** [comment]
@jywarren  please review.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Divy123) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -225,8 +225,19 @@ If only one module is to be added, `modules` is simply the name of the module.
 If multiple images are to be added, `modules` is an array, which holds the names of modules
 to be added, in that particular order.
 
+<<<<<<< HEAD
```
Comment (jywarren) [inline README.md]: Oops missed a rebase conflict! Just this change and we should be good! 👍

Comment (Divy123) [reply README.md]: Sorry for that! Forgot to save the file. Sometimes auto-save is remembered! :smile: 
Thanks

**Comment by jywarren** [comment]
I'm so sorry, one more thing - you've included two files in `/test_outputs/` -- can you remove them? Would you mind adding that folder to `.gitignore` as well? It's a new system, sorry about that!!!

**Comment by Divy123** [comment]
@jywarren please review.

**Comment by jywarren** [comment]
Great work!!!

**Pull request merged**